src/main/java/by/custom/utilcalculator/service/MessagesCreator.java

```diff
@@ -22,22 +35,34 @@ public static MessagesCreator getInstance() {
     public String buildNextStepQuestion(final UserProgress userProgress) {
```

Comment:
Maybe it's also possible to store it in tree

Reply:
Can we call somehow a method from the a tree? If it is possible, it is better to replace.